File: pascal/CaveEscape1.pas
program GameMain;
uses SplashKit, sysUtils;

procedure Main();
begin
  OpenWindow('Cave Escape', 432, 768);

  repeat // The game loop...
    ProcessEvents();
    ClearScreen(ColorWhite);
    RefreshScreen(60);
  until WindowCloseRequested('Cave Escape');
end;

begin
  Main();
end.

File: c/cave_escape_1.cpp
#include <stdio.h>
#include "splashkit.h"

int main()
{
    open_window("Cave Escape", 432, 768);

    do // The game loop...
    {
      process_events();
      clear_screen(COLOR_WHITE);
      refresh_screen(60);
    } while(!window_close_requested("Cave Escape"));

    return 0;
}
